<launch>

    <node pkg="docking" type="docking_ctrl" name="docking_ctrl" output="screen">
        <rosparam file="$(find docking)/config/params.yaml" command="load" />
    </node>

</launch>